docs/sources/guides/whats-new-in-v7-0.md

```diff
@@ -12,8 +12,184 @@ weight = -17
+This major release of Grafana contains a lot of new features and enhancements including the next step in our Observability story, powerful new features for manipulating, transforming and doing math on data and the first version of Usage Analytics in Grafana Enterprise.
+
+The Grafana 7.0 stable will be released on the 18th of May. In the meantime if your want to know more about what we have been up to and what is coming sign up to our online GrafanaCon conference.
 
 [{{< figure src="/assets/img/blog/GrafanaCONline.jpg" max-width="800px" lightbox="false" caption="GrafanaCONline May 13-29" >}}](https://grafana.com/about/events/grafanacon/2020/?source=blog)
+
+The main highlights are:
+
+- [**New Panel Editor** Redesign based on community feedback.]({{< relref "#new-panel-editor" >}})
+- [**Explore:** New tracing UI and support for visualizing Jaeger and Zipkin traces.]({{< relref "#new-tracing-ui" >}})
+- [**Enterprise** Usage Analytics and Presence.]({{< relref "#grafana-enterprise" >}})
+- [**Transformations** Transformations and simple Math operations for all data sources.]({{< relref "#transformations" >}})
+- [**Field overrides** Automatically configure panels with data from queries.]({{< relref "#field-overrides" >}})
+- [**Table** New Table panel.]({{< relref "#table-panel" >}})
+- [**Plugins** New plugins platform.]({{< relref "#plugins-platform" >}})
+- [**Tutorials** New tutorials section.]({{< relref "#new-tutorials" >}})
+- [**Cloudwatch** Support for Cloudwatch Logs in Explore and the Logs panel.]({{< relref "#cloudwatch-logs" >}})
+- [**Breaking change** PhantomJS removed.]({{< relref "#breaking-change-phantomjs-removed" >}})
+- [**Time zones** Time zone support]({{< relref "#time-zone-support" >}})
+
+## New panel editor
+
+In Grafana 7.0 we have redesigned the UI for editing panels. The first visible change is that we have separated panel display settings to a right-hand side pane that you can collapse or expand depending on what your focus is on. With this change we are also introducing our new unified option model and UI for defining data configuration and display options. This unified data configuration system powers a consistent UI for setting data options across visualizations and making all data display settings data driven and overridable.
+
+## New tracing UI
+
+With Grafana 7.0 we are adding major support for distributed tracing. This adds an important telemetry mode to complement the existing support for metrics and logs. Traces allow you to follow how single requests travelled through a distributed system. We are starting with an integrated trace viewer and two new built-in data sources: Jaeger and Zipkin.
+
+You can use the new trace view in Explore either directly and search for a particular trace or you can configure Loki to detect trace IDs in the log lines and link directly to a trace timeline pulled from Jaeger or Zipkin data source.
```

Comment:
```suggestion
You can use the new trace view in Explore either directly tosearch for a particular trace, or you can configure Loki to detect trace IDs in the log lines and link directly to a trace timeline pulled from Jaeger or Zipkin data source.
```

Reply:
Missing a space here - will fix it manually.